png.sh

```diff
@@ -4,18 +4,5 @@
 for svg in $(find svg -name "*.svg")
 do
   echo "$svg"
-  # Convert svg/.../file.svg to png/.../file.png
```

Comment:
Please retain the PNG conversion. This will be required for the Power BI plugin, etc. We want the NPM repo to have the PNGs